samples/TempLCD/GroveLCDLib.js

```diff
@@ -0,0 +1,163 @@
+// Copyright (c) 2016, Intel Corporation.
+// JavaScript library for the Grove LCD
+
+export function GroveLCD() {
```

Comment:
@jimmy-huang  this is basically breaking up the current I2CBMP280.js, I'm reusing the code from it. I'd be more than happy to create a new Lib for the other monitor I have, but that will be at another time.